tests/topotests/bgp_srv6l3vpn_sid/r1/vpnv6_rib.json

```diff
@@ -0,0 +1,169 @@
+{
+  "vrfName": "default",
+  "tableVersion": 2,
+  "routerId": "1.1.1.1",
```

Comment:
Please, replace to something non-public routable (Cloudflare does not like this).

Reply:
Done.